docs/source/common_issues.rst

```diff
@@ -821,3 +821,59 @@ This is best understood via an example:
 To get this code to type check, you could assign ``y = x`` after ``x`` has been
 narrowed, and use ``y`` in the inner function, or add an assert in the inner
 function.
+
+
+Upgrading to :pep:`585` syntax
+------------------------------
+
+Prior to :pep:`585`, instances of builtin types such as ``dict``, ``list``,
```

Comment:
The mypy docs generally seem to use "built-in" over "builtin" when it's used as an adjective

```suggestion
Prior to :pep:`585`, instances of built-in types such as ``dict``, ``list``,
```